packages/headless-styles/src/components/Button/buttonCSS.scss

```
@@ -1,67 +1,82 @@
-.btnBase {
-  composes: pandoBtn from '../shared/button.module.css';
-}
-
-.btnDefaultBorder {
-  composes: pandoBtnBorder from '../shared/button.module.css';
-}
-
-.btnDefaultRadius {
-  composes: pandoBtnRadius from '../shared/button.module.css';
-}
+@use '../shared/button' as shared;
```

We can kill this namespace since you are using it anymore. I'm not seeing a reason to add the extra load to create a new variable scope?

```suggestion
@use '../shared/button';
```

---

Yeah, we can do that.  `@extend` doesn't use the namespace, so we don't need it unless we add variables/mixins later, and we can change it then.